libs/execution_context/unit/execution_context_unit_test.cpp

```diff
@@ -16,6 +16,64 @@
 #include <optional>
 #include <set>
 
+// Define the JIT and INTERPRETER related IOCTL structs here for testing, as they
+// will not be defined in ebpf_protocol.h when the JIT and INTERPRETER are disabled.
```

Comment:
Why do you need them defined here when the functions don't exist?
Can't you simply use struct _ebpf_operation_header as the request and reply structs in the test?

Reply:
Removed these, and using `_ebpf_operation_header` now.